WalletWasabi/BitcoinCore/CoreNode.cs

```diff
@@ -90,8 +91,11 @@ public static async Task<CoreNode> CreateAsync(CoreNodeParams coreNodeParams, Ca
 				EndPointParser.TryParse($"{rpcHost}:{rpcPort}", coreNode.Network.RPCPort, out EndPoint rpce);
 				coreNode.RpcEndPoint = rpce;
 
-				var rpcClient = new RPCClient($"{authString}", coreNode.RpcEndPoint.ToString(coreNode.Network.DefaultPort), coreNode.Network);
-				coreNode.RpcClient = new RpcClientBase(rpcClient);
+				var rpcClient = new RPCClient(
+					$"{authString}",
+					coreNode.RpcEndPoint.ToString(coreNode.Network.DefaultPort),
+					coreNode.Network);
+				coreNode.RpcClient = new CachedRpcClient(rpcClient, new MemoryCache(new MemoryCacheOptions { SizeLimit = 1_000 }));
```

Comment:
As this is a default always used component I wonder if a disable flag is something to consider, or perhaps it might be enough to set the `MemoryCache.SizeLimit` to  zero.

Reply:
Yes, I think a flag could make sense here.